pkgs/test_core/pubspec.yaml

```diff
@@ -32,4 +32,4 @@ dependencies:
   # properly constrains all features it provides.
   matcher: ">=0.12.5 <0.12.6"
   # Use an exact version until the test_api package is stable.
-  test_api: 0.2.6
+  test_api: 0.2.7
```

Comment:
does this also need a dep override? or just going to submit on red?

Reply:
This was an oversight.